docs/cli.md

```diff
@@ -352,7 +352,11 @@ gcli column list --metalake metalake_demo --name catalog_postgres.hr.departments
 ```bash
 gcli table details --metalake metalake_demo --name catalog_postgres.hr.departments --audit
 ```
+#### Show tables distribution information
 
+```bash
+gcli table details --metalake metalake_demo --name catalog_postgres.hr.departments --distribution
+```
```

Comment:
I'm curious why do we need a new `--distribution` argument to display the distribution of the table, also for `audit`? Intuitively, `details` should show everything including audit, distribution, and others. IMO, it is not necessary to add arguments one by one to support displaying different attributes. This will potentially lead to too many arguments.

Reply:
Hi, @justinmclean 
Would it make more sense to include the distribution information within the table details instead of creating a separate --distribution argument?

Reply:
I'm still concerned about too many flags for different information about table, so how about fileset, topic, and potentially model, etc, looks like we may have many flags, which is not user-friendly from my point.

How about adding just one flag `--extended`, by default only basic information of the table will be shown, if `--extended` is enabled, then additional information like audit, distribution, and index will also be shown for `details` command.

Reply:
Added an `--extended` or similar option (`--all` would perhaps be a better name?) would not solve the backwards/forward compatibility issue and limits flexibility. Users might want specific details rather than everything in the "extended" or "all" output, especially if they’re scripting and only need one or two specific pieces of information.

We could add an --extended/--all for quick, high-level use, showing all additional details, but we also offer optional, specific flags like --audit, --distribution, and --index. This way users who need fine control can selectively display only what they need, while casual users can rely on --extended/--all to see more information. This approach minimises the risk of breaking user scripts in the future if new fields are added.

Filesets and topics are entities, new entities, in general, would not require any extra flags. The current design doc and existing PR topics do add a topic flag, but combining it with the name flag would be possible. Currently, Fileset has some flags as part of the fileset creation, but they will be removed and become properties. So we can avoid adding extra flags for these entities.